source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,86 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB provides clients the ability to perform write operations in bulk.
```

Comment:
Name-dropping the method name in this first chunk of text would align your formatting with our other examples and avoid burying the lede in terms of answering the question "how do i perform bulk operations"